apps/backend/src/libs/modules/server-application/base-server-application.ts

```diff
@@ -58,6 +62,8 @@ class BaseServerApplication implements ServerApplication {
 		this.app = Fastify({
 			ignoreTrailingSlash: true,
 		});
+
+		this.app.decorate(FastifyDecorator.API_VERSIONS, new Set<string>());
```

Comment:
Why do we need this?

Reply:
I think there should be a possibility to get only the endpoint from the request (like /auth/sign-in), we can use it instead